DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: cmdstanr
 Title: R Interface to 'CmdStan'
-Version: 0.5.0
-Date: 2022-03-17
+Version: 0.5.1
```

Comment:
Bumped the minor version. I could do .9000 but I was thinking we should release this temporary change to the draws method as 0.5.1

Reply:
Yeah, .1 seems good to me.